sheetstorm/tests/test_integrity_generate_test_data.py

```diff
@@ -0,0 +1,228 @@
+from django.core import management
+from django.test import TestCase
+from parameterized import parameterized
+
+from managers.models import Project
+from sheetstorm.management.commands.generate_test_data import ProjectType
+from users.factories import UserFactory
+from users.models import CustomUser
+
+
+class CreateUsersTests(TestCase):
+    def test_command_should_create_superuser_if_there_is_request(self):
```

Comment:
Try always name test methods in this convention:

"test_that_command_should_create_superuser_when_superuser_is_requested"

I mean that you should try to think to name test methods like this:

"test_that_..._when_...."